STM32/Libraries/USBprint/Src/USBprint.c

```diff
@@ -22,8 +26,12 @@ int USBnprintf(const char * format, ... )
     len += vsnprintf(&buffer[len], sizeof(buffer) - len, format, args);
     va_end (args);
 
+#ifdef USE_COMMON_USB_CDC
```

Comment:
Good temporary work around :) Looking forward to seeing only "include usb_cdc_fops.h" in there soon.